platform-api/src/main/java/io/datacater/core/stream/KafkaStreamsAdmin.java

```diff
@@ -24,15 +24,15 @@
  *
  * <p>Optional connection options: - value.deserializer.schema - schema.registry.url
  *
- * <p>Optional config options: - auth... (TODO) - key.format (default: json) - num.partitions
- * (default: 3) - replication.factor (default: 1) - value.format (default: json)
+ * <p>Optional config options: - key.format (default: json) - num.partitions (default: 3) -
```

Comment:
We need to update the JavaDoc here (rename `num.partitions` to `partition.count`.